org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -22,51 +22,51 @@ <h2>Snapshot Build @qualified.bundle.version@ (@build.date@)</h2>
 
 <h3>New Features</h3>
 <ul>
-  <li>Limit configuration of Maven check goal and Ant check element now also
-      supports percentage values 
+  <li>JaCoCo now comes with a simple command line interface
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/525">#525</a>).</li>
+  <li>Limit configuration of Maven <code>check</code> goal and Ant
+      <code>check</code> element now also supports percentage values 
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/534">#534</a>).</li>
-  <li>Exclude from a report enum methods <code>valueOf</code> and <code>values</code>
-      that are created by compiler
+  <li>Manifests of JAR files now have <code>Automatic-Module-Name</code> for Java 9
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/565">#565</a>).</li>
+  <li>Maven aggregated reports will now also include modules of <code>runtime</code>
+      and <code>provided</code> dependencies
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/498">#498</a>,
+      <a href="https://github.com/jacoco/jacoco/issues/572">#572</a>).</li>
+</ul>
+
+<p>
+  JaCoCo reports now filter out various compiler generated artifacts which
+  would lead to partial or missed coverage:
+</p>
+
+<ul>
+  <li>Methods <code>valueOf</code> and <code>values</code> of enum types
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/491">#491</a>).</li>
-  <li>Exclude from a report a part of bytecode that compiler generates for a
-      synchronized statement
+  <li>Private empty no-argument constructors
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/529">#529</a>).</li>
+  <li>Methods annotated with <code>@lombok.Generated</code> to better integrate
+      with future versions of Lombok. Initial analysis and contribution by Rüdiger zu Dohna
```

Comment:
why "future versions of Lombok"? according to https://projectlombok.org/changelog annotation is there since 1.16.14 and we did end-to-end test with 1.16.16

Reply:
Fixed. Was confused by JavaDoc: https://projectlombok.org/api/lombok/Generated.html